bevy_kayak_ui/src/render/mod.rs

```diff
@@ -23,17 +23,17 @@ mod ui_pass_driver;
 pub mod unified;
 
 pub mod node {
-    pub const UI_PASS_DEPENDENCIES: &str = "ui_pass_dependencies";
-    pub const UI_PASS_DRIVER: &str = "ui_pass_driver";
+    pub const UI_PASS_DEPENDENCIES: &str = "kayak_ui_pass_dependencies";
```

Comment:
This isn't really related, but this could cause issues if someone has the same name so I named them more exact.